### ISSUE TITLE
[Enhancement]: podcast default sort order

### ISSUE THREAD

**Issue by hoggatt** [advplyr/audiobookshelf#1454]
### Describe the feature/enhancement

Unfortunately, there's currently no way of sorting a podcast to see the most recently published first if you have episodes without a publish date. 

It would be nice if "sort by publish date" always put unlabeled podcasts at the bottom, or at least did that by default. If you sort from oldest to newest you see the tagged podcasts, but this is not a desired behavior for most people. 

In other words: can we have unlabeled podcasts count as "really old" in the sort order rather than "really new"?

**Comment by advplyr** [comment]
Updated in [v2.2.19](https://github.com/advplyr/audiobookshelf/releases/tag/v2.2.19)
